src/views/podcasts.hbs

```diff
@@ -0,0 +1,25 @@
+{{#section "head"}}
+<title>{{{__ "podcasts_title"}}} – {{{__ "common_title"}}}</title>
+{{/section}}
+
+{{#section "title"}}
+<div class="container my-4">
+  <div class="row">
+    <div class="col border-bottom">
+      <h1 class="h1">{{{__ "podcasts_title"}}}</h1>
+    </div>
+  </div>
+</div>
+{{/section}}
```

Comment:
I thought we had a template for that 🤔

Reply:
You are right...had to search for it. For some reason we are not using it though. Seems to me like it doesn't work well with our i18n options. We only use it in places where there are no translations (yet). The team page is an example:
`{{> main/title title="Wir sind Die Lernwerkstatt"}}`

Reply:
@jarjan any inputs?

Reply:
Perhaps I created this component, but didn't refactored eveywhere.